packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md

```diff
@@ -2,138 +2,98 @@
-
-**In general, how can I get familiar with the Okta product?**
-
-To get started, check out the [Okta Help Center](https://help.okta.com/) or our [OktaInc](https://www.youtube.com/user/OktaInc) channel on YouTube. App Partners are eligible for live Okta 101 sessions as well. Please email <developers@okta.com> if you are interested.
-
-## SCIM Technical Questions
-
-**Our API is similar to SCIM, but is not 100% compliant. Can we still integrate with Okta?**
-
-Unfortunately, your app's SCIM server API must be fully SCIM compliant in order to integrate with Okta.
-
-Okta SCIM client endpoints are hard coded into a template which adheres directly to [the SCIM spec](http://www.simplecloud.info/).
+### SWA
 
-**How should I be managing authentication to my SCIM API?**
+**Q: What is Secure Web Authentication (SWA)?**
 
-Okta recommends using the [OAuth 2.0 Authorization Code grant flow](/docs/guides/implement-auth-code/). Okta doesn't support the Client Credentials or Resource Owner Password Credentials Authorization grant flows for SCIM. The Authorization Code grant flow is more common in SaaS and cloud integrations and is also more secure. In addition to OAuth, Okta also supports connections with basic auth and header token auth options.
-
-**I have a multi-tenant app. How do I allow my customers to customize their specific tenant in Okta?**
-
-Use the OAuth 2.0 Authorization Code grant flow, so that you know exactly which token the customer is using.
-
-Another option is when the customer configures your app in Okta, Okta can prompt them to add their unique subdomain for your app.
-
-Okta can use part of this URL in the SCIM endpoint for that customer, for example `http://www.company.com/tenantA/scim` or `http://www.company.com/tenantB/scim`. This subdomain field can be configured in consultation with Okta after you submit your app for Okta review.
-
-![Example SCIM endpoint with subdomain](/img/oin/scim-scalar.png "Example SCIM endpoint with subdomain")
+SWA was developed by Okta to provide single sign-on for apps that do not support federated sign-on methods. Users can still sign in directly through the application and then enter their credentials for these apps on their Okta homepage. These credentials are stored so that users can access their apps with a single sign-on. When users first sign on to a SWA app from their homepage, they see a pop-up message asking if they were able to sign on successfully.
 
-**Why do I need to implement the type attribute for attributes such as emails/phoneNumbers/addresses?**
+**Q: I am creating a SWA app using the App Wizard but my application has fields on the login page in addition to the standard username and password (for example: CustomerID, OrgID). Can an app with additional fields on the login be configured using the App Wizard?**
```

Comment:
I am creating a SWA app using the App Wizard but my application has fields on the **sign-in** page in addition to the standard username and password (for example: CustomerID, OrgID). Can an app with additional fields on the **sign-in page** be configured using the App Wizard?**